File: game_server/ServerProtocol/src/Common/BagItemUseEnum/EBagItemUse_HeroDrawShowType.java
package Common.BagItemUseEnum;

/*********
 * 背包使用道具-大臣获得展示类型
 **/
public enum EBagItemUse_HeroDrawShowType {
	NONE, //0 ==== 
	POWER, //1 ==== 实力
	;
public static final EBagItemUse_HeroDrawShowType[]  EBagItemUse_HeroDrawShowType_Values = EBagItemUse_HeroDrawShowType.values();
public static final int EBagItemUse_HeroDrawShowType_Length = EBagItemUse_HeroDrawShowType_Values.length;
public static EBagItemUse_HeroDrawShowType EBagItemUse_HeroDrawShowType_FromInt(int _ivalue) {
	if(_ivalue < 0 || _ivalue >= EBagItemUse_HeroDrawShowType_Length){ return null; }
	return EBagItemUse_HeroDrawShowType_Values[_ivalue];
}
}

File: game_client/Assets/Scripts/ClientProtocol/Common/BagItemUseEnum/EBagItemUse_HeroDrawShowType.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using ALBasicProtocolPack;
using UnityEngine;


namespace Common.BagItemUseEnum
{

/// <summary>
/// 背包使用道具-大臣获得展示类型
/// </summary>
public enum EBagItemUse_HeroDrawShowType {
	NONE, //0 ==== 
	[InspectorName("POWER - idx[1] - 实力")]
	POWER, //1 ==== 实力
}

public class EBagItemUse_HeroDrawShowTypeComparer : IEqualityComparer<EBagItemUse_HeroDrawShowType>{
	public bool Equals(EBagItemUse_HeroDrawShowType x, EBagItemUse_HeroDrawShowType y) { return x == y; }
	public int GetHashCode(EBagItemUse_HeroDrawShowType obj) { return obj.GetHashCode(); }
	public static int g_iEnumCount = 2;
}
}

